packages/auth/src/graphql/schema.graphql

```diff
@@ -95,7 +95,9 @@ type Grant implements Model {
   "Wallet address of the grantee's account."
   client: String!
   "Details of the access provided by the grant."
-  access: [Access!]!
```

Comment:
I think we can keep it as `[Access!]!`, if there is no access it can simply be an empty array.